### PR TITLE
Adding missing entries in Russian translation

### PR DESCRIPTION
**Pull request by 5bentz**

### CODE REVIEW

File: src/_locales/ru/messages.json [hunk 1]
```diff
@@ -65,6 +71,21 @@
   "passff_prefs_tab_adding": {
     "message": "Добавление паролей"
   },
+  "passff_prefs_tab_misc": {
+    "message": "Разное"
+  },
+  "passff_prefs_mark_fillable": {
+    "message": "Отметить поля, которые можно заполнить, иконкой PassFF"
+  },
+  "passff_prefs_submit_fillable": {
```
Comment (5bentz) [inline src/_locales/ru/messages.json]: Translation missing I guess!

File: src/_locales/ru/messages.json [hunk 2]
```diff
@@ -170,7 +200,13 @@
   "passff_newpassword_errors_unexpected_error": {
     "message": "Извините, произошла неожиданная ошибка"
   },
-  "passff_errors_unexpected_error": {
-    "message": "Извините, произошла неожиданная ошибка. Убедитесь что ваши установки правильны и что нативная программа установлена в операционной системе."
+  "passff_errors_exec_host_app": {
```
Comment (5bentz) [inline src/_locales/ru/messages.json]: Here too.

**Comment by 5bentz** [comment]
How have you added a commit to my pull request? This is handy.

File: src/_locales/ru/messages.json [hunk 3]
```diff
@@ -65,6 +71,21 @@
...
+  },
+  "passff_prefs_submit_fillable": {
+    "message": "Submit when selected via the input menu"
+  },
+  "passff_prefs_handle_http_auth": {
+    "message": "Handle HTTP authentication"
+  },
+  "passff_prefs_directories_first": {
+    "message": "List directories first in the toolbar menu"
+  },
```
Comment (5bentz) [inline src/_locales/ru/messages.json]: Several translations missing I guess!

File: src/_locales/ru/messages.json [hunk 4]
```diff
@@ -170,7 +200,13 @@
...
-  "passff_errors_unexpected_error": {
-    "message": "Извините, произошла неожиданная ошибка. Убедитесь что ваши установки правильны и что нативная программа установлена в операционной системе."
+  "passff_errors_exec_host_app": {
+    "message": "Connection to the host app failed or returned an unexpected result!"
+  },
+  "passff_errors_upgrade_host_app": {
+    "message": "Make sure you have the latest version of the PassFF host app installed by following the {installation instructions on GitHub}."
+  },
+  "passff_no_entries_found": {
+    "message": "No entries matching this URL."
```
Comment (5bentz) [inline src/_locales/ru/messages.json]: Here too.

**Comment by swarnk** [comment]
i just copy commit URL in comment window. Github did all the magick for me.

**Comment by 5bentz** [comment]
Thank you, too much auto-magic for me!
You can update your pull request on passff's repository.

**Pull request merged**